scripts/microbe_stage/microbe_editor/microbe_editor.as

```diff
@@ -175,6 +181,20 @@ class MicrobeEditor{
 
             GetEngine().GetEventHandler().CallEvent(event);
         }
+
+        newName = playerSpecies.genus + " " + playerSpecies.epithet;
+        membrane = playerSpecies.speciesMembraneType;
+        colour = playerSpecies.colour;
+        GenericEvent@ event = GenericEvent("MicrobeEditorActivated");
+        NamedVars@ vars = event.GetNamedVars();
+
+        vars.AddValue(ScriptSafeVariableBlock("name", newName));
+        vars.AddValue(ScriptSafeVariableBlock("membrane", int(membrane)));
+        vars.AddValue(ScriptSafeVariableBlock("colour_r", int(colour.X * 255)));
```

Comment:
Would it conserve the information better if this was sent as a float?
Also this is the other side where the renaming should be done. `colourR`

Reply:
For RGB we don't really want floats, so it is intentionally truncated.

Reply:
I'd prefer if the conversion happened as close to using the inaccurate values as possible (ie. in the GUI).